rllib/agents/pg/tests/test_pg.py

```diff
@@ -22,11 +22,12 @@ def tearDown(self):
     def test_pg_compilation(self):
         """Test whether a PGTrainer can be built with both frameworks."""
         config = pg.DEFAULT_CONFIG.copy()
-        config["num_workers"] = 0
-        num_iterations = 2
+        config["num_workers"] = 1
+        config["rollout_fragment_length"] = 500
+        num_iterations = 1
 
         for _ in framework_iterator(config):
-            trainer = pg.PGTrainer(config=config, env="CartPole-v0")
+            trainer = pg.PGTrainer(config=config, env="FrozenLake-v0")
```

Comment:
Maybe loop through both CartPole and FrozenLake as envs

Reply:
+1

Reply:
done